Documentation/operations/upgrade.rst

```diff
@@ -154,6 +154,13 @@ command line or by committing the values to a YAML file:
 
 .. include:: ../gettingstarted/k8s-install-download-release.rst
 
+To minimize datapath disruption during the upgrade, the
+``upgradeCompatibility`` option should be set to the initial Cilium
+version which was installed in this cluster. Valid options are:
+
+* ``1.7`` if the initial install was Cilium 1.7.x or earlier.
```

Comment:
I guess the  "or earlier" part here could be interpreted as 1.6. Would that be a valid use-case?

Reply:
Yes, that would be valid.